<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QWidget" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #514D4D;</string>
   </property>
   <widget class="QWidget" name="MainPageBar" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>730</width>
      <height>50</height>
     </rect>
    </property>
    <widget class="QWidget" name="ProfileInfoWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>250</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #707070;
border-radius: 25px;

font-family: 'Inter SemiBold';
font-size: 18px;

color: #FFFFFF;

</string>
     </property>
     <widget class="QLabel" name="NameBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>190</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Privileges">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ExitButtonHolder" native="true">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>0</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #707070;
border-radius: 25px;
</string>
     </property>
     <widget class="QPushButton" name="ExitButton1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent; 
border: 0px;
font-family: 'Inter SemiBold';
font-size: 18px;

color: #FFFFFF;
</string>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ExitButton2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #D9D9D9;
border-radius: 25px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="MainPageField" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>730</width>
      <height>420</height>
     </rect>
    </property>
    <widget class="QWidget" name="BranchSection" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>130</width>
       <height>420</height>
      </rect>
     </property>
     <widget class="QLabel" name="BranchesLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: 'Inter SemiBold';
font-size: 18px;

color: #FFFFFF;

</string>
      </property>
      <property name="text">
       <string>Branches:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="Branches" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>130</width>
        <height>270</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #373333;
border-radius: 10px;
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StartButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(204, 218, 185, 128);
border-radius: 15px;

font-family: 'Inter SemiBold';
font-size: 18px;
line-height: 22px;

color: rgba(50, 50, 50, 128);

</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StopButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(255, 217, 217, 128);
border-radius: 15px;

font-family: 'Inter SemiBold';
font-size: 18px;
line-height: 22px;

color: rgba(50, 50, 50, 128);

</string>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EditMPButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(217, 217, 217, 128);
border-radius: 15px;

font-family: 'Inter SemiBold';
font-size: 18px;
line-height: 22px;

color: rgba(50, 50, 50, 128);

</string>
      </property>
      <property name="text">
       <string>Edit modpack</string>
      </property>
     </widget>
     <widget class="QPushButton" name="EditServerButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>390</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>This feature isn't available yet</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgba(217, 217, 217, 128);
border-radius: 15px;

font-family: 'Inter Semibold';
font-size: 18px;
line-height: 22px;

color: rgba(50, 50, 50, 128);

</string>
      </property>
      <property name="text">
       <string>Edit server</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ConsoleSection" native="true">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>590</width>
       <height>420</height>
      </rect>
     </property>
     <widget class="QLabel" name="ConsoleLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: 'Inter SemiBold';
font-size: 18px;
line-height: 22px;

color: #FFFFFF;

</string>
      </property>
      <property name="text">
       <string>Live console:</string>
      </property>
     </widget>
     <widget class="QLabel" name="BuilInfoLabel">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>0</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: 'Inter SemiBold';
font-size: 18px;
line-height: 22px;

color: rgba(255, 255, 255, 128);


</string>
      </property>
      <property name="text">
       <string>Build: 25.03/01</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="Console" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>590</width>
        <height>390</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #2D2A2A;
border-radius: 10px;
</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>349</y>
         <width>571</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="ConsoleUsername">
          <property name="styleSheet">
           <string notr="true">font-family: 'Inter SemiBold';
font-size: 15px;
line-height: 17px;

color: #CDCDCD;

</string>
          </property>
          <property name="text">
           <string>l</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ConsoleInput">
          <property name="styleSheet">
           <string notr="true">font-family: 'Inter SemiBold';
font-size: 15px;
line-height: 17px;

color: #CDCDCD;

</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
